tests/test_module_uper.py

```diff
@@ -0,0 +1,30 @@
+import asn1tools
+
+
+def main():
+  #compile a specification with 2 modules inside
+  spec = asn1tools.compile_file('tests/files/Spec.asn', 'uper')
+  
+  #encode with types from two different modules specified in the same file Spec.asn
+  enc_1 = spec.encode('ASequence',{'priority:'high', 'src':1,'dst':2,'num':0,'length':256})
+  enc_2 = spec.encode('SeqMod2',{'isvalide':True, 'name':'toto', 'ID':5)
```

Comment:
Should `ID` be renamed to `identity`, as the member is called in the specification?

Reply:
yes. Apparently I forgot to change that too.